doc/contributing/maintaining-cjs-module-lexer.md

```diff
@@ -0,0 +1,54 @@
+# Maintaining cjs-module-lexer
+
+The [cjs-module-lexer](https://github.com/nodejs/node/tree/master/deps/cjs-module-lexer)
+dependency is used within the Node.js ESM implementation to detect the
+named exports of a CommonJS module.
+
+It is used within
+[.../lib/internal/modules/translators](https://github.com/nodejs/node/blob/master/lib/internal/modules/esm/translators.js)
+in which both `internal/deps/cjs-module-lexer/lexer.js` and
+\`internal/deps/cys-module-lexer/dist/lexer.js\` are required and used.
```

Comment:
```suggestion
\`internal/deps/cjs-module-lexer/dist/lexer.js\` are required and used.
```
Unsure if this is a typo, although is there a reason the backticks are escaped here to prevent an inlined code block?

And what is the `internal` supposed to mean here? There's no directory called `internal` at the root level, which can make this a little bit confusing.

Reply:
@VoltrexMaster thanks for catching that, added some additional detail on how I think it works in terms of the paths/resolution.